developer-guide/concepts-and-designs/sig-devops/devops_observability.md

```diff
@@ -0,0 +1,22 @@
+## Expose Pipiline metrics to Prometheus
+
+​	In order to have a more intuitive statistically view about pipelines. we may need to collect some metrics about pipelines, and then expose an endpoint with metrics where a Prometheus Server can scrape.
```

Comment:
we need？

Reply:
yes, and this work is what the Jenkins Prometheus Plugin does.